blocks/identity-block/features/header-account-action/default.jsx

```diff
@@ -110,10 +109,10 @@ const HeaderAccountAction = ({ customFields }) => {
               href={manageAccountURL}
               text={phrases.t('identity-block.manage-account')}
             />
-            <DropDownLinkListItem
-              href={logoutURL}
+            <DropDownListButton
               onClick={handleLogout}
               onKeyDown={handleLogout}
```

Comment:
As this is now a `button` the `onKeyDown` event is not required

Reply:
cool can remove. this is what we want though, right? for the element to be a button not a link?

Reply:
updated

Reply:
I'd say no - as we are navigating the user to another URL, so it's a link